packages/shared/src/components/profile/SquadsList.tsx

```diff
@@ -60,8 +60,8 @@ function SquadItem({
       showLogin({
         trigger: AuthTriggers.JoinSquad,
         options: {
-          onLoginSuccess: joinSquad,
-          onRegistrationSuccess: joinSquad,
+          onLoginSuccess: () => joinSquad(),
```

Comment:
Function was always sending `(user) => unknown` but now TS was complaining when I updated correct type in `AuthModal`

Reply:
Do we need the return anywhere? if not, maybe instead we can remove the return

Reply:
Wanted to keep it as is that if we need to return we don't have to change all implementations again. Its not really doing anything.